tools/scenario-geojson2scn.py

```diff
@@ -0,0 +1,240 @@
+import argparse
+import json
+import os
+import unicodedata
+from datetime import timedelta
+from typing import List
+
+uas_id = 0
+area_id = 0
+
+# TODO comment code
+
+defaults = {
+    "zoom": 16,
+    "duration": 30,
+    "plugins": ["dronetraf", "usepelogger"],
+    "setup": [
+        "dronetraf area traffic_area",
+        "asas on",
+        "reso mvp",
+        "rmethh spd",
+        "usepeconflog on",
+        "ff",
+    ],
+    "close": ["op", "quit"],
+}
+
+
+def write_output(path, out, config):
+    if config["dry_run"]:
+        print(f"\n - {path}:")
+        print("\n".join(out))
+    else:
+        with open(path, "w") as f:
+            f.write("\n".join(out))
+
+
+def normalize_id(id):
+    # TODO is the unicode normalization really necessary?
+    id = unicodedata.normalize("NFKD", id).encode("ascii", "ignore")
+    id = "-".join(id.decode("ascii").split())
+    return id
+
+
+def get_id(feature):
+    global area_id
+
+    if "id" in feature:
+        id = feature["id"]
+    elif "name" in feature["properties"]:
+        id = normalize_id(feature["properties"]["name"])
+    else:
+        id = f"Area{area_id}"
+        area_id = area_id + 1
+    return id
+
+
+def write_cmd(cmd: str, *arguments: List[str], timestamp: timedelta = timedelta(seconds=0)):
+    hours, remainder = divmod(timestamp.seconds, 3600)
+    minutes, seconds = divmod(remainder, 60)
+    timestamp = "{:02}:{:02}:{:02}".format(int(hours), int(minutes), int(seconds))
+    return f'{timestamp}> {cmd.upper()} {" ".join(str(arg).upper() for arg in arguments)}'
+
+
+def write_aircrafts(config, features):
+    def get_point_id(feature, idx):
+        return f"{get_id(feature)}_{idx}"
+
+    def get_line_points(feature):
+        points = list(p for p in feature["geometry"]["coordinates"])
+        for idx, point in enumerate(points):
+            if len(point) > 2:
+                points[idx] = [*reversed(point[0:2]), point[2]]
+            else:
+                points[idx] = [*reversed(point[0:2]), None]
+        return points
+
+    config["ac_files"] = []
+    for feature in features:
+        props = feature["properties"]
+
+        if feature["geometry"]["type"] == "LineString" and "model" in props:
+            aid = get_id(feature)
+            pid = get_point_id(feature, 0)
+            if "start" in props:
+                start = normalize_id(props["start"])
+            else:
+                start = pid
+
```

Comment:
`end` needs to be defined here as well.
We could allow leaving out this property in the `.geojson` if the drone is supposed to take off and land at the same location.
```suggestion
                start = pid
            if "end" in props:
                end = normalize_id(props["end"])
            else:
                end = start

```